pytest_html/resources/main.js

```diff
@@ -110,21 +109,21 @@ function sort(items, key_func, reversed) {
 
 function key_alpha(col_index) {
     return function(elem) {
-        return elem.childNodes[col_index].firstChild.data.toLowerCase();
+        return elem.childNodes[1].childNodes[col_index].firstChild.data.toLowerCase();
     };
 }
 
 function key_num(col_index) {
     return function(elem) {
-        return parseFloat(elem.childNodes[col_index].firstChild.data);
+        return parseFloat(elem.childNodes[1].childNodes[col_index].firstChild.data);
     };
```

Comment:
@davehunt the problem was that before we had just one child (like, it was <td><tr></tr></td>, but now it is <tbody><td>first one about the row</td><td>logs</td></tr>), so there was one more level to access.